app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,7 @@
+<div id="copyrights">
+    <div class="container">
+      <p>
+        &copy; Copyrights <strong>Elearning</strong>. All Rights Reserved
+      </p>
+    </div>
+</div>
```

Comment:
Change from Spaces: 2 to Spaces: 4

Reply:
![image](https://user-images.githubusercontent.com/35232348/114860751-fb76b380-9e1e-11eb-8135-b74c9878a5b5.png)